--- a/bin/js-yaml.js
+++ b/bin/js-yaml.js
@@ -23,11 +23,16 @@
 
 var cli = new argparse.ArgumentParser({
   prog:     'js-yaml',
-  version:  require('../package.json').version,
   addHelp:  true
 });
 
 
+cli.add_argument('-v', '--version', {
+  action: 'version',
+  version: require('../package.json').version
+});
+
+
 cli.addArgument([ '-c', '--compact' ], {
   help:   'Display errors in compact mode',
   action: 'storeTrue'
